### PR TITLE
Provide more specific guidelines on change notes

### PR DESCRIPTION
**Pull request by mkondratek**
Pls let me know if anything more comes to your mind in this scope. 

I thought I would be good to have such guidelines and be consistent about them.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -273,6 +273,9 @@ Change notes should only be added as new bullet points listed under `Unreleased`
 Change notes should contain only the **user-facing** plugin alterations like new features, public-requested bug fixes, etc. <br/>
 In spite of the above, change notes can sometimes contain a general description of the work done in the new version
 if the alternative is leaving them empty, which should be avoided.
+The change notes should be in present tense (e.g. use `Add...` instead of `Added...`);
+they should be described in full sentences and end with a period.
+Special phrases like action name references should be highlighted with `<i>...</i>`.
```
Comment (mkondratek) [inline CONTRIBUTING.md:278]: It could be `` `...` `` but I think `<i>` tag looks better

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -273,6 +273,9 @@ Change notes should only be added as new bullet points listed under `Unreleased`
 Change notes should contain only the **user-facing** plugin alterations like new features, public-requested bug fixes, etc. <br/>
 In spite of the above, change notes can sometimes contain a general description of the work done in the new version
 if the alternative is leaving them empty, which should be avoided.
+The change notes should be in present tense (e.g. use `Add...` instead of `Added...`);
```
Comment (PawelLipski) [inline CONTRIBUTING.md:276]: Okay so we're changing the convention from now on right?

Comment (PawelLipski) [reply CONTRIBUTING.md:276]: To more git-commit-like

**Review by PawelLipski — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (mkondratek) [reply CONTRIBUTING.md:276]: Looking at the former change notes I cannot say there was any convention. 
We can go for past tense - probably it is more reasonable as in contrast to commit messages, the change notes are supposed to be read by humans (users in particular, and I do not consider developers to be humans).

Comment (mkondratek) [reply CONTRIBUTING.md:276]: @PawelLipski wdyt?

Comment (PawelLipski) [reply CONTRIBUTING.md:276]: > and I do not consider developers to be humans
:uuu:

> We can go for past tense 
Yeah, let's keep as it is

Comment (mkondratek) [reply CONTRIBUTING.md:276]: > Yeah, let's keep as it is

this order is not clear to me

Comment (PawelLipski) [reply CONTRIBUTING.md:276]: Sorry ;p keep past tense

**Pull request merged**